experiments/compositions/composition/expanders.mk

```diff
@@ -4,18 +4,9 @@
 
 ##@ Protos
 
-PROTOC ?= /usr/bin/protoc
-.PHONY: protoc
-protoc: $(PROTOC) ## Download protoc if necessary
-$(PROTOC):
-	sudo apt install -y protobuf-compiler
-	protoc --version
-	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
-	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
The removal of the `sudo apt install` command is a significant improvement. It makes the build process more hermetic and removes the need for root privileges, which is a security best practice.